tools/jdbc/src/main/java/org/duckdb/DuckDBResultSetMetaData.java

```diff
@@ -43,7 +43,9 @@ public DuckDBResultSetMetaData(int param_count, int column_count, String[] colum
 	public static DuckDBColumnType TypeNameToType(String type_name) {
 		if (type_name.startsWith("DECIMAL")) {
 			return DuckDBColumnType.DECIMAL;
-		} else {
+		} else if (type_name.equals("TIMESTAMP WITH TIME ZONE")) {
+			return DuckDBColumnType.TIMESTAMP_WITH_TIME_ZONE;
+        } else {
```

Comment:
Formatting?  Feel free to add Java stuff to our `.editorconfig` file.

Reply:
Let's see if a vim plugin can cure this formatting pain ...